### PR TITLE
Night Elf Markings Rework + 1 New Marking

### PR DESCRIPTION
**Pull request by ercarp**
## Changelog:
- Hand-painted every Night Elf marking from scratch in Blender.
  - I created the first iterations of the markings "blindly" in 2D using Photoshop, so they had a lot of errors that were not intended. The new markings were painted directly on the face of the character model, so they should be more detailed and accurate.
- Added 1 new marking, **Night Shadow** (`nightelf_markings_09`).

### nightelf_markings_01 (Bear)

<details><summary>Click to expand</summary>

⭐ **New**
image

**Old**
image

**Reference**
image

</details>

### nightelf_markings_02 (Blades)

<details><summary>Click to expand</summary>

⭐ **New**
image

**Old**
image

**Reference**
image

</details>

### nightelf_markings_03 (Crane)

<details><summary>Click to expand</summary>

⭐ **New**
image

**Old**
image

**Reference**
image

</details>

### nightelf_markings_04 (Leaf)

<details><summary>Click to expand</summary>

⭐ **New**
image

**Old**
image

**Reference**
image

</details>

### nightelf_markings_05 (Claws)

<details><summary>Click to expand</summary>

⭐ **New**
image

**Old**
image

**Reference**
image

</details>

### nightelf_markings_06 (Wings)

<details><summary>Click to expand</summary>

⭐ **New**
image

**Old**
image

**Reference**
image

</details>

### nightelf_markings_07 (Serpent)

<details><summary>Click to expand</summary>

⭐ **New**
image

**Old**
image

**Reference**
image

</details>

### nightelf_markings_08 (Owl)

<details><summary>Click to expand</summary>

⭐ **New**
image

**Old**
image

**Reference**
image

</details>

### nightelf_markings_09 (Shadow)

<details><summary>Click to expand</summary>

image

**Reference**
image

</details>

## Known issues:

- ~~`nightelf_marking_06` is showing some red artifacts. Decal needs to be resaved using different settings.~~
  - Fixed in d66ef258ade94ef9232f69c0c03b834413a9a8f3.

## In-game screenshots:

<details><summary>Click to expand</summary>

**Bear**
image
image

**Blades**
image
image

**Crane**
image
image

**Leaf**
image
image

**Claws**
image
image

**Wings**
image
image

**Serpent**
image
image

**Owl**
image
image

**Shadow**
image
image

</details>

## Tests:
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:
- Compare the Night Elf markings used on `master` to the new ones on `night-elf-markings-rework`.
  - You may either view them directly in the **Ruler Designer** or **[Debug Portrait Editor](https://ck3.paradoxwikis.com/Console_commands#Enabling_debug_mode)** or by looking up **_Female_** characters with the **_Night Elf_** trait in-game using the **Character Finder**.
- Provide feedback on the reworked markings `1-8`, and the new `nightelf_markings_09`.

### CODE REVIEW

**Comment by ValianBlue** [comment]
_¡Fantástico!_

_The leaf markings are the clear standout for me as a definite improvement, but all the markings look better than their previous iterations._

_My only suggestion is to detach the colors for the markings from the hair color._

**Comment by ercarp** [comment]
> _My only suggestion is to detach the colors for the markings from the hair color._

In what way? Because we used to have them not linked, but they generated some really strange-looking elves that didn't look good at all. In comparison, basing the color of their markings on the color of their hair looked a hundred times better.

The other option would be to create nine color variations for each marking (the amount they have in WoW), which would tally up to 81 decals if I'm doing my math right. And that's just not a feasible amount considering the maximum amount of decals you can have is supposedly 100.

Either way, it's not really up to me. I suppose we could try the first option again and see if they look as bad as I remember, but there was a reason we decided to link them to hair color instead.

**Comment by ValianBlue** [comment]
_Hmm, perhaps it's possible to set a list of hex value colors in code for the markings. I think it's doable since it's currently tied to hair color, but I may be wrong._

_If not, then perhaps it could be a player-only feature._

**Comment by zumbak04** [comment]
> Hmm, perhaps it's possible to set a list of hex value colors in code for the markings. I think it's doable since it's currently tied to hair color, but I may be wrong.

I don't think there's another way to get variety except ting it to some other color.

> If not, then perhaps it could be a player-only feature.

Markings? All females in night elven society have them.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: localization/english/wc_portraits_l_english.yml [hunk 1]
```diff
@@ -129,6 +129,7 @@
  facial_nightelf_markings_06:0 "Night Wings"
  facial_nightelf_markings_07:0 "Night Serpent"
  facial_nightelf_markings_08:0 "Night Owl"
+ facial_nightelf_markings_09:0 "Night Shadow"
```
[the remark this answers is not in the document]
Comment (ercarp) [reply localization/english/wc_portraits_l_english.yml]: My bad. Moved them in dcc99821c7c46eba9a689b816c7543dd92327a5d.

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Review by Darknight367 — APPROVED** [review]
Beautiful!

**Review by CantTouchThisJinTao — APPROVED** [review]
Pretty sure the new ones all look better. The old leaf shown in the screenshot I believe is a bugged one, didn't used to look that separated.
Tested in game on a few other races and looked decent as well.
Not sure if it matters but Tyrande has one of the new markings instead of her custom one.

Approved :)

**Comment by ercarp** [comment]
> The old leaf shown in the screenshot I believe is a bugged one, didn't used to look that separated.

It's not bugged, I made the old markings so I know it looked like that. It just wasn't noticeable in-game because the separated parts were hidden by their eyebrows which I disabled for the preview images here.

**Pull request merged**